app/controllers/anonymous_feedback/export_requests_controller.rb

```diff
@@ -36,20 +37,25 @@ def export_request_params
   end
 
   def anonymous_feedback_params
-    clean_paths
-    @anonymous_feedback_params ||= params.permit(:from, :to, :organisation, :path, paths: []).to_h
+    @anonymous_feedback_params ||= params.permit(:from, :to, :organisation, :path_set_id).to_h
   end
 
-  def clean_paths
-    if params[:path]
-      params[:paths] = [params[:path]]
-    elsif params[:paths] && params[:paths].instance_of?(String)
-      params[:paths] = params[:paths].split(",").map(&:strip)
-    end
+  def saved_paths
+    @saved_paths ||= Support::Requests::Anonymous::Paths.find(anonymous_feedback_params[:path_set_id])
+  end
+
+  def paths
+    return [] if anonymous_feedback_params[:path_set_id].blank?
+
+    saved_paths.try(:paths)
```

Comment:
Do we need to use `try` or can be just do `.paths`? I'd imagine now that we don't support `params[:paths]`, we'd want it to error if the `saved_paths` doesn't exist.

Reply:
What was the resolution to this?